### PR TITLE
Simple navigation

### PR DESCRIPTION
**Pull request by quantum-noise**
Navigation between 'My Posts', 'Feed' (own + subscribers posts), 'Top Posts' (by num of likes).

### CODE REVIEW

File: app/services/post.service.js [hunk 1]
```diff
@@ -1,18 +1,36 @@
...
+    getPostsByAccount: getPostsByAccount,
+    getPostById: getPostById,
+    getTop: getTop
   }
 
-  function getAllPosts() {
+  function getTop() {
     return $http
-      .get('/api/v1/post/')
+      .get('/api/v1/post/?order_by=like')
```
Comment (alekseyC64) [inline app/services/post.service.js]: pass it as params object. `$http.get(url, {params: your params})`

File: app/services/post.service.js [hunk 2]
```diff
@@ -1,18 +1,36 @@
...
+      .then(function(response) {
+        return response.data.objects
+      });
+  }
+
+  function getFeed() {
+    return $http
+      .get('/api/v1/post/?feed&order_by=date')
```
Comment (alekseyC64) [inline app/services/post.service.js]: the same, pass params in more convenient way https://code.angularjs.org/1.6.2/docs/api/ng/service/$http

File: app/services/post.service.js [hunk 3]
```diff
@@ -1,18 +1,36 @@
...
       .then(function(response) {
         return response.data.objects;
       });
   }
 
+    function getPostsByAccount(accountId) {
+    return $http
+      .get('/api/v1/post/?order_by=date&account=' + accountId)
```
Comment (alekseyC64) [inline app/services/post.service.js]: the same

File: app/services/post.service.js (hunk 1 above)
Comment (quantum-noise) [inline app/services/post.service.js]: done

File: app/services/post.service.js (hunk 2 above)
Comment (quantum-noise) [inline app/services/post.service.js]: done

File: app/services/post.service.js (hunk 3 above)
Comment (quantum-noise) [inline app/services/post.service.js]: done

File: app/app.js [hunk 4]
```diff
@@ -1,6 +1,8 @@
 import 'bootstrap/dist/css/bootstrap.css';
 import './templates/common/common.css';
 import FeedController from './controllers/feed.controller';
+import MyController from './controllers/my.controller';
```
Comment (NikitaSedyx) [inline app/app.js]: weird name `MyController`
imo `MyPostsController` is much better

File: app/controllers/my.controller.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+export default function MyController(postService, Session) {
+
+    postService.getPostsByAccount(Session.user.id)
+    .then(posts => {
```
Comment (NikitaSedyx) [inline app/controllers/my.controller.js]: nit: indentation before `then` please

File: app/app.js (hunk 4 above)
Comment (quantum-noise) [inline app/app.js]: What about path /my ?

File: app/services/post.service.js [hunk 6]
```diff
@@ -1,18 +1,47 @@
 export default function postService($http) {
 
   return {
-    getAllPosts: getAllPosts,
-    getPostById: getPostById
+    getFeed: getFeed,
```
Comment (NikitaSedyx) [inline app/services/post.service.js]:
```
return {
  getFeed,
  getPostsByAccount,
  ...
}
```

File: app/config/routes.js [hunk 7]
```diff
@@ -38,5 +38,23 @@ export default function routes($stateProvider, $urlRouterProvider) {
       data: {
         isAuthenticated: true,
       }
+    })
+    .state('my', {
```
Comment (NikitaSedyx) [inline app/config/routes.js]: state name is weird too :)

File: app/services/post.service.js [hunk 8]
```diff
@@ -1,18 +1,47 @@
...
+    getPostById: getPostById,
+    getTop: getTop
   }
 
-  function getAllPosts() {
+  function getTop() {
     return $http
-      .get('/api/v1/post/')
+      .get('/api/v1/post/',
+        {params: {
```
Comment (NikitaSedyx) [inline app/services/post.service.js]: don't you think it's better to make variable for `params`?

File: app/app.js (hunk 4 above)
Comment (NikitaSedyx) [inline app/app.js]: there is comment below

File: app/config/routes.js [hunk 9]
```diff
@@ -38,5 +38,23 @@ export default function routes($stateProvider, $urlRouterProvider) {
...
+      url: '/my',
+      template: require('../templates/Feed.html'),
+      controller: 'MyController',
+      controllerAs: 'vm',
+      data: {
+        isAuthenticated: true,
+      }
+    })
+    .state('top', {
```
Comment (NikitaSedyx) [inline app/config/routes.js]: same with this one
why not `topPosts`

File: app/app.js (hunk 4 above)
Comment (quantum-noise) [inline app/app.js]: done

File: app/controllers/my.controller.js (hunk 5 above)
Comment (quantum-noise) [inline app/controllers/my.controller.js]: done

File: app/services/post.service.js (hunk 6 above)
Comment (quantum-noise) [inline app/services/post.service.js]: done

File: app/config/routes.js (hunk 7 above)
Comment (quantum-noise) [inline app/config/routes.js]: done

File: app/services/post.service.js (hunk 8 above)
Comment (quantum-noise) [inline app/services/post.service.js]: done

File: app/config/routes.js (hunk 9 above)
Comment (quantum-noise) [inline app/config/routes.js]: done

File: app/services/post.service.js [hunk 10]
```diff
@@ -1,18 +1,51 @@
...
   }
 
-  function getAllPosts() {
+  function getFeed() {
+    var params = {};
+    params.order_by = byDate;
+    params.feed = '';
     return $http
-      .get('/api/v1/post/')
+      .get('/api/v1/post/', {params})
```
Comment (alekseyC64) [inline app/services/post.service.js]: `/api/v1/post/` repeats several time, could you make it as a constant?

File: app/services/post.service.js [hunk 11]
```diff
@@ -1,21 +1,55 @@
 export default function postService($http) {
 
+  var byDate = '-date';
```
Comment (NikitaSedyx) [inline app/services/post.service.js]: why do you need `var` with ES6?

**Pull request merged**